apps/taxes/src/modules/taxjar/order-confirmed/taxjar-order-confirmed-payload-transformer.ts

```diff
@@ -0,0 +1,69 @@
+import { LineItem } from "taxjar/dist/util/types";
+import { OrderConfirmedSubscriptionFragment } from "../../../../generated/graphql";
+import { numbers } from "../../taxes/numbers";
+import { taxProviderUtils } from "../../taxes/tax-provider-utils";
+import { TaxJarTaxCodeMatches } from "../tax-code/taxjar-tax-code-match-repository";
+import { TaxJarConfig } from "../taxjar-connection-schema";
+import { TaxJarOrderConfirmedTarget } from "./taxjar-order-confirmed-adapter";
+import { TaxJarOrderConfirmedPayloadLinesTransformer } from "./taxjar-order-confirmed-payload-lines-transformer";
+
+export function sumPayloadLines(lines: LineItem[]): number {
```

Comment:
I dont like this method to be exported because `sumPayloadLines` doesn't say what exactly is summed up. You sum unit price but there can be also gross, net etc

Reply:
I added a comment to handle it later. While a valid comment, this is not in the scope of this PR. I just changed the name of the files.